proof_test.go

```diff
@@ -17,7 +17,7 @@ func TestTreeGetWithProof(t *testing.T) {
 	require := require.New(t)
 	for _, ikey := range []byte{0x11, 0x32, 0x50, 0x72, 0x99} {
 		key := []byte{ikey}
-		tree.Set(key, []byte(random.Str(8)))
+		tree.Set(key, []byte(RandStr(8)))
```

Comment:
what's wrong with using cmn?

Reply:
I don't think we should be importing tendermint into this repo. Its the first step of 1) switching to using an adopted error handling practice in the tendermint org 2) removing tendermint as a dependency in this repo.

Reply:
back to using cmn, but from github.com/tendermint/tm-cmn

Reply:
> I don't think we should be importing tendermint into this repo

Completely agree 👍